src/passes/LimitSegments.cpp

```diff
@@ -22,7 +22,13 @@ namespace wasm {
 
 struct LimitSegments : public Pass {
```

Comment:
Please add a comment documenting this flag, similar to

https://github.com/WebAssembly/binaryen/blob/19dd23db3c20214e2e6a73023529c2496f9c2a50/src/passes/Asyncify.cpp#L236-L240